src/client/client4.js

```diff
@@ -1489,6 +1489,13 @@ export default class Client4 {
         );
     };
 
+    testElasticsearch = async () => {
```

Comment:
Minor but is Elasticsearch really one word? Looks so odd to me, 0/5 if you want to change it